mappings/net/minecraft/world/biome/BiomeGenerator.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS none/ajk net/minecraft/world/biome/BiomeGenerator
-	FIELD a isDecorating Z
+	FIELD a decorating Z
```

Comment:
I'd like more discussion on this if possible it just doesn't really seem right to me.